### PR TITLE
Review request for mircoverse tribute project

### PR DESCRIPTION
**Pull request by mercymugambi**
This is a past project completed during the Admissions Trial. 
This pull request is an exercise that tests my skills as a code reviewer.
As code reviewer below has to be put into consideration:

1. Are the best practices in HTML code applied?
2. Are semantic HTML tags used?
3. Are CSS selectors correctly used?
4. is CSS box model used?
5. Are linters and work flow used?

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,196 @@
+*
+{
+    margin: 0;
+    padding: 0;
+    font-family: poppins;
```
Comment (mercymugambi) [inline style.css:5]: I think you can use font-family: "poppins", san serif; to create a sense of formality and elegance 🤗.

**Review by mercymugambi — COMMENTED** [review]
Hi @mercymugambi 👋
Good Job so far! 👍  However, there are some issues you need to work on to perfect on this. 
## Highlights 💯
✅ Semantic rules perfectly applied.
✅ Commit messages well written.
✅Title and description are detailed. 
## Required Changes ♻️

1. I think you should implement workflows and linters in your project . This will help you diagnose and fix bugs 😄 [linters guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md), [workflows](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/git_github_basics.md).
2. Project code looks good as it follows html and css best practises👏. However, i suggest you document the project by adding README file. This will enable anyone looking at your project understand the project better. To add a proffessional readme file you can use the guidelines on this link; [guide for writting proffesional readme](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md).

<i>Check other comments under the code review too. </i>👇🏽
soclose

## Optional suggestions

<i>N/A
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.</i>

<i>Cheers and Happy coding!👏👏👏</i>

<i>Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.</i>

<i><b>Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.<i/>

File: style.css (hunk 1 above)
Comment (iambenkis) [reply style.css:5]: Kindly use some emoji and GIF smile to make your comments more friendly :hugs:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @mercymugambi   :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: The student phrased the comments politely
:heavy_check_mark: The student comments are clear and easy to understand
:heavy_check_mark: used the GitHub inline comments feature
:heavy_check_mark: 2 comments in the review.


Good job on implementing most of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Require Changes

- [ ] Great job reviewing this project, 👏 , I can not hide it, but you made wonderful job 🤙 , but you should add two inline comments to meet the requirements 💯 , kindly add another one.

- [ ] Kindly use some emoji and GIF smile to make your comments more friendly :hugs: 

_Check the comments in your comments_

# Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"="Card image cap">
+                    <div class="card-block">
+                        <p class="card-text">Tribute Page</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </main>
+
```
Comment (mercymugambi) [inline index.html:71]: The structure looks wow. However, I think there are unused lines in the projects that you can do away with; for it to look more prettier. Right?😎 To do so, install an extension called prettier or you can run the linters code to clean it for you.

**Review by jerryowusu — APPROVED** [review]
Hi @mercymugambi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
:heavy_check_mark: Nice and polite comments
:heavy_check_mark: The student comments are clear and easy to understand
:heavy_check_mark: Used GitHub inline comments feature

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**